src/trackers/ZNTH.py

```
@@ -0,0 +1,80 @@
+                console.print(f'[bold red]Porn/xxx is not allowed at {self.tracker}.')
+                if cli_ui.ask_yes_no("Do you want to upload anyway?", default=False):
+                    pass
+                else:
+                    return False
+            else:
+                return False
+
+        return should_continue
+
+    async def get_type_id(
+        self, meta: dict[str, Any], type: str = "", reverse: bool = False, mapping_only: bool = False
```

Ruff (0.15.15)

[error] Function argument `type` is shadowing a Python builtin

(A002)